playbooks/roles/tower/tasks/bootstrap_credentials.yml

```diff
@@ -38,3 +38,14 @@
     ssh_key_data: '/tmp/tower_ssh'
     ssh_key_unlock: '{{ tower_ssh_password }}'
     username: '{{ tower_ssh_user }}'
+
+- name: 'Create AWS Credentials'
+  tower_credential:
+    name: '{{ item }}.AWS'
+    description: '{{ item }} AWS Credential'
+    organization: '{{ tower_organization }}'
+    state: present
+    kind: '{{ tower_credential_bundle_aws_type }}'
```

Comment:
This variable could be set in defaults/main.yml of the role rather than something that gets read in from the credential repo. Ideally the credential repo would just store private credentials?

Reply:
yep good point